PP_APP/server/frontend/templates/base.html

```diff
@@ -2,149 +2,134 @@
 <!-- DJANGO BASE TEMPLATE STARTS HERE -->
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="">
-    <meta name="author" content="">
-
-    <title> {{ title }} | Pointless Packaging</title>
-
-    <!-- Bootstrap core CSS -->
-    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
-
-    <!-- Custom fonts for this template -->
-    <link href="{% static 'vendor/fontawesome-free/css/all.min.css' %}" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Varela+Round" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Basic&display=swap" rel="stylesheet">
-
-    <!-- Bootstrap core JavaScript -->
-    <script src="{% static 'vendor/jquery/jquery.min.js' %}" defer></script>
-    <script src="{% static 'vendor/bootstrap/js/bootstrap.bundle.min.js' %}" defer></script>
-    
-    <!-- Plugin JavaScript -->
-    <script src="{% static 'vendor/jquery-easing/jquery.easing.min.js' %}" defer></script>
-
-    <!-- Chart - Bootstrap Optional Theme -->
```

Comment:
Why was this deleted? What was edited it base.html?

Reply:
i just replace the whole file with my own version, i added some links in the base.html for my ranking table. It just easier for me to to update this base.html,  otherwise i need to go through the file line by line. and the .html is very tedious.